Add missing vector sprites and costumes to libraries

thisandagain opened the pull request:
### Resolves
Resolves GH-1423

### Proposed Changes
- Adds a variety of missing vector sprites and costumes to the libraries
- Sprites and costumes that are not included have either been replaced (e.g. musical instruments) or are scheduled to be replaced soon (e.g. spooky, vehicles, etc.)

### Browser Coverage
Check the OS/browser combinations tested (At least 2)

Mac
 * [ ] Chrome 
 * [ ] Firefox 
 * [ ] Safari
 
Windows
 * [ ] Chrome 
 * [ ] Firefox 
 * [ ] Edge
 
Chromebook
 * [ ] Chrome
 
iPad
* [ ] Safari

Android Tablet
* [ ] Chrome

ericrosenbaum commented:
Just a note for future cleanup: I noticed that all of the sprites except for one have `"rotationStyle": "normal"`. The "Avery walking" sprite has `"rotationStyle": "left-right"`. We should either just make sure they are all normal, or make all of the "walk cycle" sprites have it set to left-right.

ericrosenbaum commented:
It's a bit weird that we will have two different sprites named "Jamie" (vector basketball character) and "Jaime" (bitmap walking character). We should consider renaming one of them.

ericrosenbaum commented:
This PR seems to have replaced a newer-version princess (blue dress) with an older-version princess (pink dress). I suppose we should have both, so the newer-version princess (blue dress) needs to be re-added.

ericrosenbaum commented:
Some sprites weirdly have both "meow" and "pop" sounds.

Also at least one, the rocks sprite, has two copies of the pop sound.

ericrosenbaum commented:
This PR also reverted the new shark back to an older (cartoony) version of the shark. I think we should have both, so the older (cartoony) shark should be re-added.

ericrosenbaum commented:
We should consider consolidating some sprites where they are similar variations, such as the scarfs and shoes.

thisandagain commented:
Additional changes from feedback:
- Remove old shark sprite / costumes
- Remove old princess sprite / costumes
- Resolve issue with two pop sounds on the rock sprite
- Ensure all sprites have one (and only one) sound
- Remove two (legacy) bitmap sprites
- Remove religious imagery pending further discussion
- Removed redundant dove (dove2)
- Removed all four penguins pending new "arctic" theme from illustrator
- Removed lioness pending new "savanah" theme from illustrator
- Removed redundant monkey (monkey1)
- Removed blocks from "Speaker" sprite
